js/core/net/tcp-timer.js

```diff
@@ -13,27 +13,19 @@
 // limitations under the License.
 
 'use strict';
-var tcpSocketState = require('./tcp-socket-state');
-var connections = new Set();
+// const tcpSocketState = require('./tcp-socket-state');
+const connections = new Set();
 
-function timeoutHandler() {
-  for (var connSocket of connections) {
-    connSocket._timerTick();
-  }
+let initTimeout;
```

Comment:
I just tried it and eslint errors that 'timeoutHandler' was used before it was defined... I could also just disable `no-use-before-define` if you prefer it strictly as a function declaration